css/styles.css

```diff
@@ -268,3 +268,71 @@ textarea {
   resize: vertical;
   height: 170px;
 }
+
+@media (min-width: 500px) {
+  body {
+    background-color: #172b4d;
+  }
+
+  .app-wrapper {
+    width: 100vw;
+    display: flex;
+    flex-direction: column;
+  }
+
+  .welcome-line {
+    background-image: url("/assets/shapes/background-shapes/desktop/desktop-header-shapes.svg");
+  }
+
+  .portfolio-cards {
+    margin: 142px;
+    gap: 142px;
+  }
+
+  .portfolio-cards .card {
+    flex-direction: row;
+  }
+
+  .portfolio-cards .card .left-block {
+    flex-direction: column;
+  }
+
+  .card .head {
+    flex-direction: column;
+  }
+
+  .image1 {
+    display: none;
+  }
+
+  .image2 {
+    display: block;
+  }
+
+  .about-myself {
+    display: flex;
+    flex-grow: 1;
+    flex-basis: 0;
+    padding-left: 142px;
+    padding-right: 142px;
+  }
```

Comment:
- [ ] Great job so far. This part is a bit different from the Figma design( CSS wise). There is a difference in spacing, please make sure that the spacing looks similar or close to the design.

### Your implementation
![image](https://user-images.githubusercontent.com/59416136/154527160-dcfe7677-6a18-461d-bef8-51b1d61c5a24.png)

### Figam's Design
![image](https://user-images.githubusercontent.com/59416136/154527274-86379230-2d48-4b01-b761-3dcfa1ad70d8.png)

Kindly fix that.